django/db/migrations/migration.py

```diff
@@ -162,8 +162,10 @@ def unapply(self, project_state, schema_editor, collect_sql=False):
                 schema_editor.collected_sql.append("--")
                 if not operation.reduces_to_sql:
                     continue
-            if not schema_editor.connection.features.can_rollback_ddl and operation.atomic:
-                # We're forcing a transaction on a non-transactional-DDL backend
+            atomic_operation = operation.atomic or (self.atomic and operation.atomic is not False)
```

Comment:
To help eliminate other variables I just tried it with `tox` (e.g., `tox -e py36-postgres -- --settings=test_postgres`) and got the same result.